tidy3d/components/data/monitor_data.py

```diff
@@ -2518,6 +2536,73 @@ def renormalize_fields(self, proj_distance: float) -> FieldProjectionAngleData:
         # compute updated fields and their coordinates
         return self.make_renormalized_data(phase, proj_distance)
 
+    @property
+    def tangential_dims(self):
+        tangential_dims = ["theta", "phi"]
+        return tangential_dims
+
+    @staticmethod
+    def _check_coords_sorted(coord: np.ndarray, name: str):
+        """Helper for checking whether an array is sorted and raises an exception if it is not."""
+        is_sorted = np.all(np.diff(coord) >= 0)
+        if not is_sorted:
+            raise ValueError(f"{name} was not provided as a sorted array.")
+
+    def _check_integration_suitability(self):
+        """Checks whether the sampling of ``theta`` and ``phi`` is suitable for
+        integrating over a spherical surface."""
+        if (
+            len(self.theta) < MIN_ANGULAR_SAMPLES_SPHERE
+            or len(self.phi) < 2 * MIN_ANGULAR_SAMPLES_SPHERE
+        ):
+            raise ValueError(
+                "There are not enough sampling points along `theta` or `phi` for accurate integration. "
+                f"Currently, {len(self.theta)} samples for `theta` and {len(self.phi)} samples for `phi`. "
+                "Consider using at least 45 samples for `theta` and 90 samples for `phi`, which "
```

Comment:
The value here looks different from `MIN_ANGULAR_SAMPLES_SPHERE = 10` defined at the beginning?